src/message/messageActionSheet.js

```diff
@@ -82,6 +86,21 @@ const reply = ({ message, dispatch, ownUser }) => {
 reply.title = 'Reply';
 reply.errorMessage = 'Failed to reply';
 
+const replyWithMention = ({ message, dispatch, ownUser, narrow, startReplyWithMention }) => {
+  const replyNarrow = getNarrowForReply(message, ownUser.user_id);
+  const replyMessage = `@**${message.sender_full_name}**`;
+  if (JSON.stringify(replyNarrow) !== JSON.stringify(narrow)) {
```

Comment:
It's better to use `isEqual` from `lodash.isequal`, rather than `JSON.stringify`.